README.md

```diff
@@ -27,3 +27,10 @@ export RANDOM_SUFFIX=""
 export RUSTFLAGS="--cfg=web_sys_unstable_apis"
 trunk serve --release --dist docs
 ```
+
+
+## Running/building from source (native)
+
+```
+cargo run -r --bin raphael-xiv
```

Comment:
Nitpick, but I prefer the full flag name (e.g. `--release` instead of `-r`) for clarity.